Automattic/wp-calypso
# Purchases: Update cancellation flow copy

This PR addresses part of #2392 by updating `ManagePurchase` and `CancelPurchase` with the copy changes from that issue.

#### Testing instructions

- Run `git checkout update/cancellation-copy` and start your server
- Apply this patch `code-D1469`

*Subscriptions*
- Open `/purchases/:site/:purchaseId` for a non-domain subscription that has auto-renew enabled (e.g. a plan).
- Assert that the link to cancel reads 'Cancel Subscription and Refund', or 'Cancel Subscription' if it is out of the refund window.
- Click the aforementioned cancel link.
- Assert that purchases which are refundable match [this screenshot](https://cloudup.com/c30EJIftlSi).
- Assert that purchases which are not refundable match [this screenshot](https://cloudup.com/cBWvH1teVxX).
- Assert that the refund amount is correct

*Domains*
- Open `/purchases/:site/:purchaseId` for a domain purchase that has auto-renew enabled (e.g. a plan)
- Assert that the link to cancel reads 'Cancel Domain and Refund', or 'Cancel Domain' if it is out of the refund window.
- Click the aforementioned cancel link.
- Assert that domains that are within the refund window match [this screenshot](https://cloudup.com/c0BiX3bKFuT) (the refund amount may differ)
- Assert that domains that are not within the refund window match [this screenshot](https://cloudup.com/c69B4sw0WXB).
- Assert that the refund amount is correct

*One time purchases*
- Open `/purchases/:site/:purchaseId` for a one time purchase (e.g. a premium theme) that was purchased within the last 30 days.
- Assert that the link to cancel reads 'Cancel and Refund'.
- Click the aforementioned cancel link.
- Assert that the cancel page that appears matches [this screenshot](https://cloudup.com/cdghzdenTTD), with different values depending on the theme.
- Assert that the refund amount is correct

#### Additional notes

There are more changes to the flow in #2392 but these will be handled in separate PRs.

#### Reviews
